packages/orchestrator-ui-components/src/contexts/PolicyContext.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC, ReactNode, createContext, useCallback } from 'react';
+
+export type Policy = {
+    isAllowed: (resource: string, routerPath: string) => boolean;
+};
+
+export const PolicyContext = createContext<Policy>({ isAllowed: () => true });
+
+export type PolicyProviderProps = {
+    initialIsAllowed: (resource: string, routerPath: string) => boolean;
+    children: ReactNode;
+};
+
+export const PolicyContextProvider: FC<PolicyProviderProps> = ({
+    initialIsAllowed,
+    children,
+}) => {
+    const isAllowed = useCallback(initialIsAllowed, [initialIsAllowed]);
```

Comment:
I guess `initialIsAllowed` is because of naming issue.

Might consider changing `isAllowed` to `isAllowedCallback` instead.

Reply:
The callback is removed and i renamed the prop to isAllowedHandler to show a clear difference between the handling part and the usePolicy hook